data/component-design-tokens/callout-design-tokens.json

```
@@ -1,134 +1,218 @@
-    "value": "{color.info.default}",
+  "callout-h-success-heading-color": {
+    "value": "{color.text.light}",
     "type": "color"
   },
-  "callout-info-icon-color": {
-    "value": "{color.greyscale.white}",
+  "callout-h-emergency-heading-color": {
+    "value": "{color.text.light}",
     "type": "color"
   },
-  "callout-info-border-color": {
-    "value": "none",
+  "callout-h-info-border-color": {
+    "value": "{color.info.border}",
     "type": "color"
   },
-  "callout-warning-color-bg-content": {
-    "value": "{color.greyscale.100}",
+  "callout-l-info-border-color": {
+    "value": "{color.info.border}",
     "type": "color"
   },
-  "callout-warning-color-bg-statusbar": {
-    "value": "{color.warning.default}",
+  "callout-h-important-border-color": {
+    "value": "{color.warning.border}",
     "type": "color"
   },
-  "callout-warning-icon-color": {
-    "value": "{color.greyscale.black}",
+  "callout-l-important-border-color": {
+    "value": "{color.warning.border}",
     "type": "color"
   },
-  "callout-warning-border-color": {
-    "value": "none",
+  "callout-h-success-border-color": {
+    "value": "{color.success.border}",
     "type": "color"
   },
-  "callout-success-color-bg-content": {
-    "value": "{color.greyscale.100}",
+  "callout-l-success-border-color": {
+    "value": "{color.success.border}",
     "type": "color"
   },
-  "callout-success-color-bg-statusbar": {
+  "callout-h-emergency-border-color": {
+    "value": "{color.emergency.border}",
+    "type": "color"
+  },
+  "callout-l-emergency-border-color": {
+    "value": "{color.emergency.border}",
+    "type": "color"
+  },
+  "callout-info-heading-bg-color": {
+    "value": "{color.info.border}",
+    "type": "color"
+  },
+  "callout-h-info-heading-bg-color": {
+    "value": "{color.info.default}",
+    "type": "color"
+  },
+  "callout-important-heading-bg-color": {
+    "value": "{color.warning.light}",
+    "type": "color"
+  },
+  "callout-h-important-heading-bg-color": {
+    "value": "{color.warning.default}",
+    "type": "color"
+  },
+  "callout-success-heading-bg-color": {
+    "value": "{color.success.light}",
+    "type": "color"
+  },
+  "callout-h-success-heading-bg-color": {
     "value": "{color.success.default}",
     "type": "color"
   },
-  "callout-success-icon-color": {
+  "callout-emergency-heading-bg-color": {
+    "value": "{color.emergency.light}",
+    "type": "color"
+  },
+  "callout-h-emergency-heading-bg-color": {
+    "value": "{color.emergency.default}",
+    "type": "color"
+  },
+  "callout-info-content-bg-color": {
     "value": "{color.greyscale.white}",
     "type": "color"
   },
-  "callout-success-border-color": {
-    "value": "none",
+  "callout-h-info-content-bg-color": {
+    "value": "{color.info.light}",
     "type": "color"
   },
-  "callout-emergency-color-bg-content": {
-    "value": "{color.greyscale.100}",
+  "callout-l-info-content-bg-color": {
+    "value": "{color.info.light}",
```

This should be color.info.background
```suggestion
    "value": "{color.info.background}",
```